.github/workflows/release.yaml

```diff
@@ -13,8 +13,20 @@ jobs:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           fetch-depth: 0
+          
+      - name: Set tag output
```

Comment:
I think we can drop this step now that we aren't using this output parameter anymore

Reply:
Yeah

Reply:
I think once we drop this step, we should be good to go here.